### PR TITLE
Compose improvements (v3)

### PR DESCRIPTION
**Pull request by cshawaus**
The compose project was in need of a refreshment and it is now ready for more eyes to start looking over the changes to verify they meet the needs for the archetype.

Related PR: https://github.com/aem-design/aemdesign-aem-core/pull/33

**What is covered:**
- Moved all webpack configurations to [@aem-design/compose-webpack](https://github.com/aem-design/npm-compose-webpack)
- Migrated Sass code into an ITCSS structure
- Improved the core JavaScript so it is clearer
- Fixed ESLint rules and presets/plugins
- Renamed Styleguide to DLS
- Revamped the DLS content structure
- Cleaned up the content generator structure
- Added generators for colours and icons
- Removed Cypress due to implementation and maintenance complexities

Todos:
- [ ] Finish DLS content
- [ ] Update groovy specs

### CODE REVIEW

**Comment by cshawaus** [comment]
@wildone Can you please help shed some light on why the `mvn clean` task in ci is failing? Can't really see any reason for it to not be working as it was before.

**Comment by cshawaus** [comment]
@wildone New issue, SonarQube is looking for the now renamed **source** folder but I can't find where this reference is set.

**Comment by wildone** [comment]
wow, it is magnificent!

**Comment by wildone** [comment]
wow, compiles so fast!

File: aemdesign-aem-showcase/src/main/content/jcr_root/content/aemdesign-showcase/au/en/component/lists/page-list/.content.xml [hunk 1]
```diff
@@ -1,937 +1,47 @@
...
+          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          jcr:primaryType="cq:Page">
     <jcr:content
-        cq:tags="[aemdesign-showcase:content-type/page]"
-        cq:template="/conf/aemdesign-showcase/settings/wcm/templates/twocolumn"
+        cq:allowedTemplates="[/conf/aemdesign-showcase/settings/wcm/templates/.*]"
+        cq:template="/conf/aemdesign-showcase/settings/wcm/templates/onecolumn"
         jcr:primaryType="cq:PageContent"
-        jcr:title="Page List"
+        jcr:title="Generic Example Six"
```
Comment (wildone) [inline aemdesign-aem-showcase/src/main/content/jcr_root/content/aemdesign-showcase/au/en/component/lists/page-list/.content.xml:9]: is this right? did we move rest of the content to another page?

**Pull request closed without merging**